conf/dbs.config

```
@@ -60,10 +60,16 @@ params {
     // Boltz links
     boltz_ccd_link           = 'https://huggingface.co/boltz-community/boltz-1/resolve/main/ccd.pkl'
     boltz_model_link         = 'https://huggingface.co/boltz-community/boltz-1/resolve/main/boltz1_conf.ckpt'
+    boltz2_aff_link          = 'https://huggingface.co/boltz-community/boltz-2/resolve/main/boltz2_aff.ckpt'
+    boltz2_conf_link         = 'https://huggingface.co/boltz-community/boltz-2/resolve/main/boltz2_conf.ckpt'
+    mols_link                = 'https://huggingface.co/boltz-community/boltz-2/resolve/main/mols.tar'
```

For consistency, shouldn't we rename it?
```suggestion
    boltz2_conf_link         = 'https://huggingface.co/boltz-community/boltz-2/resolve/main/boltz2_conf.ckpt'
    boltz2_mols_link         = 'https://huggingface.co/boltz-community/boltz-2/resolve/main/mols.tar'
```